index.html

```diff
@@ -13,6 +13,50 @@
   </head>
 
   <body>
-   
+    <div class="container">
+      <header>
+        <div class="header">
+          <div class="header-p">
+            <p class="name">Kevin Toro</p>
+          </div>
+          <div class="fa-bars-icon">
+            <i class="fa-solid fa-bars"></i>
+          </div>
+        </div>
+      </header>
+      <main>
+        <section class="section-1">
+          <div class="headline">
+            <div class="Primary-text">Hey There. I’m Toro</div>
```

Comment:
This is like the Main Heading of the page for the Search Engines. For SEO and semantic purposes, you have to make it `h1`.